iperf/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM clearlinux:latest
+MAINTAINER long1.wang@intel.com
+
+ARG swupd_args
+
+RUN swupd update $swupd_args && \
```

Comment:
Please add "--no-boot-update" as parameter to "swupd update".
The update on container doesn't need update the boot partition.